### DISCUSSION TITLE
Meaning of SignerOutput

### DISCUSSION THREAD

**Discussion by ghost** [trustwallet/wallet-core#1977] [Q&A]
Hi, this is my first time using trust wallet core
Can tell me the meaning of SignerOutput when I call AnySigner.sign? 
How can I transfer tokens, check if the transaction is error from SignerOutput

**Comment by ghost** [comment]
@hewigovens  Please help me, thanks.

**Reply by optout21** [reply]
SignerOutput contains the prepared transaction, including signature, in a form that can be used directly in an RPC call for braodcasting (or almost in the form, detail detail by blockchain).

**Reply by ghost** [reply]
So if I want to send token with web3j from Android app, I will use this web3j.ethSendRawTransaction(SignerOutput).sendAsync().get().
Is that what you mean? @catenocrypt
